src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
+export const TitleWrapper = styled.div`
+    display: flex;
+    gap: 12px;
+    align-items: center;
+    justify-self: right;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        gap: 8px;
+        margin-top: 8px;
+        justify-self: left;
+    }
+`;
+
+export const StyledVideoIcon = styled(VideoIcon)`
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        width: 17px;
+        height: 17px;
+    }
+`;
+
+export const TitleText = styled.div`
+    font-family: poppins;
+    font-weight: 500;
+    font-size: 0.8125rem;
+    font-size: clamp(0.8125rem, 0.675rem + 0.6875vw, 1.5rem);
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        font-size: 13px;
+    }
+`;
+
+export const ButtonsWrapper = styled.div`
```

Comment:
I would change it to: 

![image](https://user-images.githubusercontent.com/117176251/220740250-b1f10edb-7ec9-43b6-9c5e-3eccc896879d.png)

`export const ButtonsWrapper = styled.div`
  display: flex;
  gap: 16px;

  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
    gap: 12px;
    margin-left: 30px;
  }
`;`

`margin-left: 30px;` makes it not overlay page title